terraform/staging/variables.tf

```diff
@@ -1,5 +1,11 @@
+variable "create_secrets_first" {
```

Comment:
I could potentially have used `initial_create` here, but felt it was cleaner to keep that focused solely on the stuff needed for DNS setup etc.